tools/ops/org-pr-monitor.py

```diff
@@ -89,6 +90,12 @@ def _format_metric(self, key, value, unit, dimensions):
             d['Dimensions'].append({"Name": k, "Value": v})
         return d
 
+    def get_timestamp(self):
```

Comment:
given no compatibility concern for this one, let's just go to utc unconditionally.

Reply:
sounds good